### PR TITLE
Report back to user of jag when program can't run because of version …

### PR DESCRIPTION
**Pull request by erikcorry**
…mismatch

### CODE REVIEW

File: src/jaguar.toit [hunk 1]
```diff
@@ -51,13 +51,13 @@ main args:
   network := net.open
   socket := network.tcp_listen port
   address := "http://$network.address:$socket.local_address.port"
-  logger.info "running Jaguar device '$name' (id: '$id') on '$address'"
+  logger.info "Running Jaguar device '$name' (id: '$id') on '$address'"
```
Comment (kasperl) [inline src/jaguar.toit:54]: Please don't change the info strings here.

Comment (floitsch) [reply src/jaguar.toit:54]: What's the reason for keeping them how they are?
Style, or are they used somewhere else?

**Review by floitsch — APPROVED** [review]
LGTM with Kasper's comment addressed.

File: src/jaguar.toit (hunk 1 above)
Comment (kasperl) [reply src/jaguar.toit:54]: There are lots of other logs. They all look like 'INFO: running ...'.

**Pull request merged**